app/hub/Views/CreateAccountView/CreateAccountView.jsx

```diff
@@ -105,7 +105,7 @@ const CreateAccountView = (props) => {
 						/>
 						{confirmEmailError && (
 							<div className="CreateAccountView__inputError">
-								{t('your_emails_do_not_match')}
+								{t('your_email_do_not_match')}
```

Comment:
We probably should fix this typo in the key name, but would require fixing all the languages in Transifex.  Leave it for now and I'll make a note to fix it later.

Reply:
I changed the key to match the messages.json file.
Do you want me to change all the messages.json files?

Reply:
@christophertino sounds like you think we should punt on this?